# Mental health in the world: What does the data tell us?

### Mental health affects the world differently, but how differently? How common are mental health disorders? Are specific countries affected differently? What does the Data tell us?

#### I decided to look at data from 2023 about how common mental health disorders are around the world so I could then compare several countries. 

![Share of population with mental health disorders, 2023](https://ourworldindata.org/grapher/share-with-mental-and-substance-disorders.png)

[Share of population with mental health disorders 2023](https://ourworldindata.org/grapher/share-with-mental-and-substance-disorders)

## Looking at the data and comparing countries

#### Here are a few countries I compared

In [9]:
countries = ["United States", "United Kingdom", "Australia", "Greece"]

mental_health_rates = [18.0, 22.0, 21.7, 19.8]

print(countries)
print(mental_health_rates)

['United States', 'United Kingdom', 'Australia', 'Greece']
[18.0, 22.0, 21.7, 19.8]


#### I decided to calculate the average of the countries I selected 

In [10]:
average_rate = sum(mental_health_rates) / len(mental_health_rates)

print("Average rate:", average_rate, "%")

Average rate: 20.375 %


#### I decided to see which country in my sample had the highest percentage

In [11]:
highest_rate = max(mental_health_rates)
highest_country = countries[mental_health_rates.index(highest_rate)]

print("Highest rate in my sample:", highest_country)
print("Percentage:", highest_rate, "%")

Highest rate in my sample: United Kingdom
Percentage: 22.0 %


## What does the data tell us?

### The countries in my sample had different percentages of people with mental health disorders. The average of my selected countries was 20.375%. This shows that mental health disorders affect a high number of populations even though the percentages vary between the countries.

# Conclusion

## This small investigation reveals how data can help us understand mental health by using population statistics. Calculating an average and comparing countries allowed me to turn statistics into something easier to interpret. Statistics alone does not tell us why the countries have different rates. Measurement, diagnosis, reporting, and other factors can influence the numbers.